packages/xo-common/src/api-errors.js

```diff
@@ -172,3 +172,11 @@ export const patchPrecheckFailed = create(20, ({ errorType, patch }) => ({
   },
   message: `patch precheck failed: ${errorType}`,
 }))
+
+export const missingPatchesFetchFailed = create(21, ({ host, reason }) => ({
+  data: {
+    objectId: host,
+    reason,
+  },
+  message: 'cannot fetch missing patches',
+}))
```

Comment:
Could we find a more generic error for this?

Reply:
This time it's too generic :slightly_frowning_face: 

As we discussed the rule is: all data fields of a given error type should be always relevant.